crates/hdk/src/lib.rs

```diff
@@ -73,7 +73,7 @@
 //! The only way to execute logic inside WASM is by having the conductor (host) call a function that is marked as an `extern` by the zome (guest).
 //!
 //! > Note: From the perspective of hApp development in WASM, the "guest" is the WASM and the "host" is the running Holochain conductor.
-//!   The host is _not_ the "host operating system" in this context.
+//! > The host is _not_ the "host operating system" in this context.
```

Comment:
Clippy was upset about this

Reply:
Should this be a new bullet or a continuation of the first one? I think Clippy is wanting this indented with an extra space if it's the latter

Reply:
One more space you mean?

It is rendered as a continuation
![image](https://github.com/user-attachments/assets/51183f31-823e-45c0-971b-ee9cb9428e19)

I think in order to create a new paragraph you need a blank line in MD.

I just tried no space, two spaces and three spaces, but clippy always says no.
```
//! The host is _not_ the "host operating system" in this context.
   |     ^
   |
   = help: if this not intended to be a quote at all, escape it with `\>`
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#doc_lazy_continuation
   = note: `-D clippy::doc-lazy-continuation` implied by `-D clippy::style`
   = help: to override `-D clippy::style` add `#[allow(clippy::doc_lazy_continuation)]`
help: add markers to start of line
   |
76 | //! > The host is _not_ the "host operating system" in this context.
```